packit/upstream.py

```diff
@@ -345,74 +336,6 @@ def get_current_version(self) -> str:
         logger.debug("sanitized version = %s", ver)
         return ver
 
-    def bump_spec(
-        self,
-        version: str = None,
-        changelog_entry: str = None,
-        bump_release: bool = False,
-    ):
-        """
-        Run rpmdev-bumpspec on the upstream spec file: it enables
-        changing version and adding a changelog entry
-
-        :param version: new version which should be present in the spec
-        :param changelog_entry: new changelog entry (just the comment)
-        :param bump_release: "bump trailing .<DIGIT> component if found, append .1 if not"
-                             from the rpmdev-bumpspec --help
-        """
-        cmd = ["rpmdev-bumpspec"]
-        if version:
-            # 1.2.3-4 means, version = 1.2.3, release = 4
-            cmd += ["--new", version]
-        if changelog_entry:
-            cmd += ["--comment", changelog_entry]
-        if bump_release:
-            cmd += ["-r"]
-        cmd.append(str(self.absolute_specfile_path))
-        run_command(cmd)
```

Comment:
Ok, now when looking at it I understand your confusion. This `bump_spec()` looks like redundant, confusing crap (sorry, whoever wrote it :-) ). Glad you removed it :+1: